schemas/tests/unit/test_filings.py

```diff
@@ -44,25 +48,30 @@ def test_invalid_ar_filing():
         'filing': {
             'header': {
                 'name': 'annualReport',
-                'date': '2019-04-08'
+                'date': '2019-04-08',
+                'certifiedBy': 'full name',
+                'email': 'no_one@never.get',
             },
             'business': {
                 'cacheId': 1,
                 'foundingDate': '2007-04-08',
                 'identifier': 'CP1234567',
                 'lastLedgerTimestamp': '2019-04-15T20:05:49.068272+00:00',
                 'legalName': 'legal name - CP1234567'
+            },
+            'annualReport': {
+                'annualGeneralMeetingDate': '2018-04-08',
             }
         }
     }
     is_valid, errors = validate(iar, 'filing')
 
-    # if errors:
-    #     for err in errors:
-    #         print(err.message)
+    if errors:
+        for err in errors:
+            print(err.message)
     print(errors)
 
-    assert not is_valid
+    assert is_valid
```

Comment:
This one is trying to test invalid ("test_invalid_ar_filing"). You've switched it to a test for *valid* filing.